### PR TITLE
feat(setup.md): add command

### PR DESCRIPTION
**Pull request by kishore03109**
## Problem

Currently, we are seeding the rows manually for our e2e tests. However, writing directly to DB is generally not good eng practice and can cause problems like [Postgres's primary key sequence going out of sync](https://stackoverflow.com/questions/244243/how-to-reset-postgres-primary-key-sequence-when-it-falls-out-of-sync). 

## Solution

This PR aims to alievate that by just providing 2 commands to run instead.

## Tests 
1. run 
```
INSERT INTO sites (name, site_status, job_status, creator_id)
VALUES ('e2e email test site 2', 'INITIALIZED', 'READY', 1);
INSERT INTO repos (name, url, created_at, updated_at, site_id)
SELECT 'e2e-email-test-repo 2', 'https://github.com/isomerpages/e2e-email-test-repo-2', NOW(), NOW(), id FROM sites WHERE name = 'e2e email test site 2';
```
2. verify that rows are properly added in the `sites` and `repos` tables respectively. 
3. delete added rows from the respective tables.

### CODE REVIEW

**Review by seaerchin — APPROVED** [review]
lgtm if you tested that it works on local

**Comment by seaerchin** [comment]
can hold off on merging until monday? see if i got enough time to do this tmrw

**Comment by kishore03109** [comment]
@seaerchin yes sir

**Comment by kishore03109** [comment]
@seaerchin sir any updates ah

**Comment by seaerchin** [comment]
> @seaerchin sir any updates ah

merge bah didn't work on this

File: cypress/SETUP.md
Comment (QiluXie) [inline cypress/SETUP.md:24]: the `creator_id` in `sites` table is integer type (`int8`) instead of text/string, we should use `1` without the quote here to match the data type.

**Review by QiluXie — CHANGES_REQUESTED** [review]

File: cypress/SETUP.md
Comment (kishore03109) [reply cypress/SETUP.md:24]: sure, but just a heads up that [postgres auto converts the type for us](https://www.postgresql.org/docs/current/typeconv.html)

Comment (QiluXie) [reply cypress/SETUP.md]: Understand that in this case since text `1` can be parsed to number `1` without any ambiguity, postgres will perform the implicit conversion. But still the best practice is to use the correct type that matches the target column when it's possible.

**Review by QiluXie — APPROVED** [review]
lgtm

**Pull request merged**
